```
Error running mint automation
+ mkdir -p 11570-03c71a6/gopath/src/github.com/minio
+ git -C 11570-03c71a6/gopath/src/github.com/minio clone --quiet https://github.com/minio/minio.git
+ git -C 11570-03c71a6/gopath/src/github.com/minio/minio remote add minio https://github.com/minio/minio.git
+ git -C 11570-03c71a6/gopath/src/github.com/minio/minio fetch --quiet minio pull/11570/head:pr11570
+ git -C 11570-03c71a6/gopath/src/github.com/minio/minio checkout --quiet pr11570
+ GO111MODULE=on
+ GOPATH=/home/mint-auto/mint-auto/11570-03c71a6/gopath
+ make -C 11570-03c71a6/gopath/src/github.com/minio/minio --quiet
can't load package: package .: build constraints exclude all Go files in /home/mint-auto/mint-auto/11570-03c71a6/gopath/src/github.com/minio/minio
make: *** [build] Error 1

```

--- a/.github/workflows/go.yml
+++ b/.github/workflows/go.yml
@@ -11,7 +11,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        go-version: [1.14.x, 1.15.x]
+        go-version: [1.15.x, 1.16.x]
         os: [ubuntu-latest, windows-latest]
     steps:
       - uses: actions/checkout@v2

--- a/README_zh_CN.md
+++ b/README_zh_CN.md
@@ -89,7 +89,7 @@ service minio start
 
 ## 使用源码安装
 
-采用源码安装仅供开发人员和高级用户使用,如果你还没有Golang环境， 请参考 [How to install Golang](https://golang.org/doc/install)。最低需要Golang版本为 [go1.14](https://golang.org/dl/#stable)
+采用源码安装仅供开发人员和高级用户使用,如果你还没有Golang环境， 请参考 [How to install Golang](https://golang.org/doc/install)。最低需要Golang版本为 [go1.15](https://golang.org/dl/#stable)
 
 ```sh
 GO111MODULE=on go get github.com/minio/minio

--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/minio/minio
 
-go 1.14
+go 1.15
 
 require (
 	cloud.google.com/go v0.39.0

--- a/main.go
+++ b/main.go
@@ -1,5 +1,3 @@
-// +build go1.14
-
 /*
  * MinIO Cloud Storage, (C) 2016-2020 MinIO, Inc.
  *

